src/js/graphql_endpoint/package-lock.json

```diff
@@ -14,8 +14,14 @@
         "@grpc/grpc-js": "^1.2.2",
         "@opentelemetry/api": "^1.0.4",
         "@opentelemetry/exporter-zipkin": "^1.0.1",
+        "@opentelemetry/instrumentation-express": "^0.28.0",
```

Comment:
Should this file be in this PR?

(I ask because nothing else in this PR does anything with the `graphql_endpoint` code.)

Reply:
Probably not, but it fixes a yarn audit before I knew it was a soft fail I'm taking out a separate PR to fix all yarn audits soon. Can downgrade though.